CHANGELOG.md

```diff
@@ -1,51 +1,219 @@
+
+## [v1.1.0](https://github.com/grpc-ecosystem/go-grpc-middleware/tree/v1.1.0) (2019-09-12)
+[Full Changelog](https://github.com/grpc-ecosystem/go-grpc-middleware/compare/v1.0.0...v1.1.0)
+
+**Implemented enhancements:**
+
+- Rate Limit Interceptor  [\#134](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/134)
+- Add if the trace was sampled to the ctxTags [\#184](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/184) ([domgreen](https://github.com/domgreen))
+
+**Closed issues:**
+
+- logrus module pulled in by test [\#228](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/228)
+- grpclogger should implement LoggerV2 interface [\#219](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/219)
+- Retry timeout will overwrite the origin timeout in ctx [\#216](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/216)
+- retry not working when grpc-server close the connection [\#199](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/199)
+- Question about package naming [\#197](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/197)
+- Checksum Mismatch on go get [\#193](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/193)
+- go modules support [\#185](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/185)
+- FR: In auth package, add exclude path to Interceptor builder [\#176](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/176)
+- Replace "golang.org/x/net/context" with "context" [\#175](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/175)
+- FR: In retry package, pass context to custom retry handler [\#171](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/171)
+- FR: In recovery package, pass context to custom recovery handler [\#168](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/168)
+- Can I start auth proxy? [\#158](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/158)
+- retry doesnt seems to work  [\#154](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/154)
+- grpc\_retry:not work as a calloption in my situation [\#151](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/151)
+- Downstream interceptors are not re-executed on retry [\#110](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/110)
+- Can we get semver releases? [\#40](https://github.com/grpc-ecosystem/go-grpc-middleware/issues/40)
+
+**Merged pull requests:**
+
+- Quick Fix of \#228 [\#229](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/229) ([domgreen](https://github.com/domgreen))
+- remove last net/context reference will help fix \#175 [\#227](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/227) ([domgreen](https://github.com/domgreen))
+- Move from dep to modules [\#226](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/226) ([domgreen](https://github.com/domgreen))
```

Comment:
Can we have the old style arrangments of PRs? 😅 
Like - `- [#226](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/226) Support for go modules.`

```suggestion
- [#226](https://github.com/grpc-ecosystem/go-grpc-middleware/pull/226) Support for go modules.
```